xformers/ops/fmha/__init__.py

```diff
@@ -7,7 +7,18 @@
 
 import torch
 
-from . import attn_bias, cutlass, decoder, flash, small_k, triton, triton_splitk
+from . import (
+    attn_bias,
+    ck,
+    ck_decoder,
+    ck_splitk,
+    cutlass,
+    decoder,
+    flash,
+    small_k,
+    triton,
+    triton_splitk,
+)
```

Comment:
For making it simpler for the users (and in our tests/benchmarks), we could add a `cutlass_or_ck` alias that is either cutlass or CK depending on the GPU type.